include/taskolib/Sequence.h

```diff
@@ -667,6 +670,14 @@ class Sequence
      */
     void set_tags(const std::vector<Tag>& tags);
 
+    /**
+     * Set the automation flag.
+     *
+     * If the flag is set the sequence can be automatically triggered by an external
+     * call.
```

Comment:
Maybe we should explicitly state that Taskolib does not use this flag for anything itself, but that it is for user code to decide if autorun happens and when.

Reply:
I can change it to "This flag is only informative but it can be used by third party code to decide when the autorun happens and when."

Reply:
That sounds excellent! :+1: